.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,52 @@
+name: Bump version
+
+on:
+  workflow_call:
+    inputs:
+      tag-prefix:
+        required: false
+        default: 'v'
+        type: string
+    outputs:
+      tag:
+        description: Newly created tag
+        value: ${{ jobs.bump-version.outputs.tag }}
+
+jobs:
+  bump-version:
+    name: Bump Version on master
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    outputs:
+      tag: ${{ steps.save-tag.outputs.tag }}
+
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+
+      - name: cat package.json
+        run: cat ./package.json
+
+      - name: Automated Version Bump
+        id: version-bump
+        uses: phips28/gh-action-bump-version@master
+        with:
+          tag-prefix: ${{ inputs.tag-prefix }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This need GITHUB token? better to use https://github.com/snapshot-labs/snapshot.js/blob/master/.github/workflows/update-minor-version.yml#L17 easy to understand

Reply:
This one just open a PR. The point of this PR is to do everything all a once, at the same time.